adapters/repos/db/vector/hnsw/delete.go

```diff
@@ -174,7 +174,12 @@ func (h *hnsw) copyTombstonesToAllowList(breakCleanUpTombstonedNodes breakCleanU
 
 // CleanUpTombstonedNodes removes nodes with a tombstone and reassigns
 // edges that were previously pointing to the tombstoned nodes
-func (h *hnsw) CleanUpTombstonedNodes(stopFunc cyclemanager.StopFunc) error {
+func (h *hnsw) CleanUpTombstonedNodes(shouldBreak cyclemanager.ShouldBreakFunc) error {
+	err, _ := h.cleanUpTombstonedNodes(shouldBreak)
+	return err
+}
+
+func (h *hnsw) cleanUpTombstonedNodes(shouldBreak cyclemanager.ShouldBreakFunc) (error, bool) {
```

Comment:
return values shouldn't be (bool, error)? instead of (error, bool)?

Reply:
fixed